morpheus/loaders/__init__.py

```diff
@@ -0,0 +1,21 @@
+# Copyright (c) 2022-2023, NVIDIA CORPORATION.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""
+Morpheus loader definitions, each loader is automatically registered when imported and will be available for use.
+"""
+
+from morpheus.loaders import file_to_df_loader
+from morpheus.loaders import fsspec_loader
+
+__all__ = []
```

Comment:
This needs to contain strings with the names `file_to_df_loader`, and `fsspec_loader`

Reply:
I don't think so. Neither of those classes is meant to be used directly. Just importing them here causes them to be registered with the loader registry.